Testscripts/Windows/VERIFY-SRIOV-FAILSAFE-XDP.ps1

```diff
@@ -43,7 +42,7 @@ function Ping-XDPDump {
     }
 
     $currentStatus = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password `
-        -command "tail -1 ~/xdpdumpout$LogSuffix.txt | head -1" -runAsSudo
+        -command "tail -1 ~/xdpdumpout$LogSuffix.txt" -runAsSudo

```

Comment:
```head -1``` removed ?

Reply:
yes. As tail -1 would only return 1 line. we don't need head -1.